android/gradle.properties

```diff
@@ -43,6 +43,12 @@ newArchEnabled=false
 # If set to false, you will be using JSC instead.
 hermesEnabled=true
 
+# Enable using the next version of asynchrounous storage
+AsyncStorage_useNextStorage=true
+# Recommended version
+AsyncStorage_kotlinVersion=1.8.10
```

Comment:
The recommended version from the AsyncStorage package is `1.9.20`. I don't think we should downgrade to 1.8.10.
Or is there any reason you chose this specific version?

Source: https://react-native-async-storage.github.io/async-storage/docs/advanced/next/#configuration

Reply:
Well, it was exactly 1.8.10 when I started doing this task